### DISCUSSION TITLE
Caused by: java.io.NotSerializableException: com.google.inject.assistedinject.FactoryProvider2

### DISCUSSION THREAD

**Discussion by naiveZhou** [openTCS/opentcs#21] [Q&A] [answered]

**Comment by swltr** [comment]
@naiveZhou I only get to this now. Since you deleted the text from your post: Has the problem been resolved in the meantime?

**Comment by naiveZhou** [comment, marked as answer]
Yes,I have resolved the problem and it is not a bug. The reason for this is that OpenTCS 5.4 removed the code,but some versions exist. Thanks.
For Example:
I use the error method,so it can not initial success.



         



***@***.***
 
From: Stefan Walter
Date: 2022-05-31 22:07
To: openTCS/opentcs
CC: smallzhou; Mention
Subject: Re: [openTCS/opentcs] Caused by: java.io.NotSerializableException: com.google.inject.assistedinject.FactoryProvider2 (Discussion #21)
@naiveZhou I only get to this now. Since you deleted the text from your post: Has the problem been resolved in the meantime?
